### DISCUSSION TITLE
How to Clear Marks on Enter (Notion-like Behavior) without Custom Extensions?

### DISCUSSION THREAD

**Discussion by kenfdev** [ueberdosis/tiptap#5944] [General Questions]
Hi all,

I'm building a Notion-like editor using Tiptap and aiming to replicate a specific behavior:  
- When the user presses `Enter` to create a new paragraph, all active styles (marks) should reset.  
- When the user presses `Shift+Enter`, the styles should remain active (continuing the current line with the same formatting).  

I understand that this behavior can be implemented by adding a custom extension to handle `Enter` key events. However, I've noticed that editors like [Novel](https://novel.sh/) and [Notitap](https://sereneinserenade.github.io/notitap/) achieve this without adding any additional extension for this behavior. (afaik) 

I'm wondering if there's a more "native" or built-in way to handle this within Tiptap's existing functionality, similar to how Novel and Notitap achieve it.  

### My Questions:  

1. Does Tiptap provide a configuration or existing method to reset marks on `Enter` natively without custom extensions?  
2. If not, could someone clarify how other editors built on Tiptap (like Novel or Notitap) handle this behavior?  

Thank you in advance for your guidance!

**Comment by romancavry** [comment]
@kenfdev did you find the answer?
